core/common/src/main/java/alluxio/util/CommonUtils.java

```diff
@@ -852,5 +861,22 @@ public static boolean isAddressReachable(String hostname, int port) {
     return result;
   }
 
+  /**
+   * @return the major version of the current JVM, 8 for 1.8, 11 for java 11
+   * see https://www.oracle.com/java/technologies/javase/versioning-naming.html for reference
+   */
+  private static int initMajorVersion() {
```

Comment:
can you add one or more unit test(s) for this method

Reply:
emmm not sure how we can test this? like how do we know the ground truth, from within a jvm?

Reply:
test expected JVM inputs (`1.8.X-XXX`, `11.X.X.`, etc?) and make sure this function returns the correct output